web_src/css/helpers.css

```diff
@@ -60,6 +60,8 @@ Gitea's private styles use `g-` prefix.
   text-overflow: ellipsis;
 }
 
+.gt-max-width-24rem { max-width: 24rem !important; }
+
```

Comment:
This does not seem a general useful helper, maybe add a unique class and add it there instead?

Reply:
I think it will be useful in the future. I can see many hard-coded width/max-width.

`a unique class` it still needs one CSS name and at least one line.

Reply:
Ok, keep it.